demisto_sdk/commands/format/tests/test_formatting_yml_test.py

```diff
@@ -891,3 +891,37 @@ def test_check_for_subplaybook_usages(self, repo):
 
         playbook_data = playbook.yml.read_dict()
         assert playbook_data['tasks']['1']['task']['playbookId'] == "my-sub-playbook"
+
```

Comment:
What about a playbook and scripts? In order to be able to test those too, let's add a parameterized decorator to this function and give a different YML entity every time.